lib/compress.js

```diff
@@ -1273,7 +1273,21 @@ merge(Compressor.prototype, {
         def(AST_Constant, return_false);
         def(AST_This, return_false);
 
+        var pure_regex = /[@#]pure/;
```

Comment:
Yeah, there's always a risk of an unintentional comment collision.

I'm going to change it to `/[@#]__PURE__/`.